CONTRIBUTING.md

```diff
@@ -26,27 +31,37 @@ git checkout common/master -b <your_branch_name>
 
 6. Make your changes
 
-7. Commit yout changes
+7. Commit your changes
+8. Before push your changes into the main branch, please:
+
+- Check if your code contribution is formatted and styled accordantly with the project source code running:
+
+  ```bash
+  pre-commit run --all
+  ```
+
+> **Tip:** Ensure you have `pre-commit` tool installed by doing `python -m pip -r requirements-test.txt` inside you virtual environment.
+> **Note:** If there is any error or warning, please solve it before continue.
+
+1.  Push the committed changes to your fork (the `origin` remote)
 
-8. Push the commited changes to your fork (the `origin` remote)
 ```bash
 git push origin HEAD
 ```
 
-9. Go to your fork (on the browser), you'll a big yellow banner offering you to create a pull request to the `common` remote. Click to `Create pull request`
+10. Go to your fork (on the browser), you'll a big yellow banner offering you to create a pull request to the `common` remote. Click to `Create pull request`
```

Comment:
A hint to select our GeeksCAT fork maybe contribute to avoid errors raising PRs to the original project ^^